### PR TITLE
Updated the default namespace for new components 

### LINKED ISSUE #387: Remove Labs from the default namespace for new components
### Describe the bug

The default templated namespace for all components uses the Labs moniker, which won't work for the main toolkit.

After [reviewing past discussions](https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/4488#issuecomment-1458638688), @michael-hawker, @niels9001 and @Arlodotexe have settled on:
- `CommunityToolkit.Labs.WinUI.Controls.ComponentName` for the package ID.
- `CommunityToolkit.WinUI.Controls.ComponentName` for the default namespace.

Reasoning behind this:
- Our tooling has a major focus on UI controls (tests, samples, etc)
- Namespaces and package IDs can be customized per component anyway, giving us the flexibility to use something different later (see #386).

### Steps to reproduce

```text
n/a
```

### Expected behavior

n/a

### Screenshots

_No response_

### Code Platform

- [ ] UWP
- [ ] WinAppSDK / WinUI 3
- [ ] Web Assembly (WASM)
- [ ] Android
- [ ] iOS
- [ ] MacOS
- [ ] Linux / GTK

### Windows Build Number

- [ ] Windows 10 1809 (Build 17763)
- [ ] Windows 10 1903 (Build 18362)
- [ ] Windows 10 1909 (Build 18363)
- [ ] Windows 10 2004 (Build 19041)
- [ ] Windows 10 20H2 (Build 19042)
- [ ] Windows 10 21H1 (Build 19043)
- [ ] Windows 11 21H2 (Build 22000)
- [ ] Other (specify)

### Other Windows Build number

_No response_

### App minimum and target SDK version

- [ ] Windows 10, version 1809 (Build 17763)
- [ ] Windows 10, version 1903 (Build 18362)
- [ ] Windows 10, version 1909 (Build 18363)
- [ ] Windows 10, version 2004 (Build 19041)
- [ ] Other (specify)

### Other SDK version

_No response_

### Visual Studio Version

_No response_

### Visual Studio Build Number

_No response_

### Device form factor

_No response_

### Additional context

_No response_

### Help us help you

Yes, but only if others can assist.

### PR DESCRIPTION
**Pull request by Arlodotexe**
This PR
- Closes #387
- Changes the package id to `CommunityToolkit.Labs.WinUI.Controls.ComponentName`.
- Changes the default namespace / csproj name to `CommunityToolkit.WinUI.Controls.ComponentName`.
- Doesn't change existing components, or new components in PR.

### CODE REVIEW

File: template/ProjectTemplate/src/CommunityToolkit.WinUI.Controls.ProjectTemplate.csproj [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <Project Sdk="MSBuild.Sdk.Extras/3.0.23">
   <PropertyGroup>
     <ToolkitComponentName>ProjectTemplate</ToolkitComponentName>
-    <PackageId>CommunityToolkit.Labs.$(PackageIdVariant).ProjectTemplate</PackageId>
+    <PackageId>CommunityToolkit.Labs.$(PackageIdVariant).Controls.ProjectTemplate</PackageId>
```
Comment (Arlodotexe) [inline template/ProjectTemplate/src/CommunityToolkit.WinUI.Controls.ProjectTemplate.csproj:4]: Let's move `CommunityToolkit.Labs` to a variable supplied by Build.Directory.props

**Comment by michael-hawker** [comment]
We have some references to Toolkit.Labs.All.sln in the readme and gitignore we should clean up too.

File: template/ProjectTemplate/src/CommunityToolkit.WinUI.Controls.ProjectTemplate.csproj [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
     <Description>This package contains ProjectTemplate.</Description>
     <Version>0.0.1</Version>
 
     <!-- Rns suffix is required for namespaces shared across projects. See https://github.com/CommunityToolkit/Labs-Windows/issues/152 -->
-    <RootNamespace>CommunityToolkit.Labs.WinUI.ProjectTemplateRns</RootNamespace>
+    <RootNamespace>CommunityToolkit.WinUI.Controls.ProjectTemplateRns</RootNamespace>
```
Comment (michael-hawker) [inline template/ProjectTemplate/src/CommunityToolkit.WinUI.Controls.ProjectTemplate.csproj:9]: This is just the uno workaround and not the actual namespace used.

We should be updating the `namespace CommunityToolkit.Labs.WinUI;` in the template to `namespace CommunityToolkit.WinUI.Controls;` eh?

If you search for `.Labs.` in vscode it shows us many of the places (like in the template XAML components too) where we need to update tot his new pattern.

We should also update the `template.json` name and identity maybe too?

**Comment by Arlodotexe** [comment]
> We have some references to Toolkit.Labs.All.sln in the readme and gitignore we should clean up too.

Thanks for pointing that out - we should probably remove `Labs` from that solution name. It's referenced in a lot of places.

File: template/ProjectTemplate/samples/ProjectTemplateCustomSample.xaml [hunk 3]
```diff
@@ -3,7 +3,7 @@
       xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-      xmlns:labs="using:CommunityToolkit.Labs.WinUI"
+      xmlns:labs="using:CommunityToolkit.WinUI.Controls"
```
Comment (michael-hawker) [inline template/ProjectTemplate/samples/ProjectTemplateCustomSample.xaml:6]: Should we change `xmlns:labs` here as well?

Do we go back to old `xmlns:controls` that we kind of aligned on for old toolkit e.g. past `xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"`?

Or do we do something new like `xmlns:ctwuc` which would be accurate abbreviation, but ugly?

Thinking maybe just `xmlns:controls`? Makes it simpler for folks that upgrade from 7.x->8.0 to align conventions?

Comment (Arlodotexe) [reply template/ProjectTemplate/samples/ProjectTemplateCustomSample.xaml:6]: Intellisense would use `xmlns:controls` as the default I think? That + the above points, I'm sold on using that. I'll get on that.

**Comment by Arlodotexe** [comment]
Looks like we might be forced to implement #321 here anyway. Will continue tomorrow.

**Comment by michael-hawker** [comment]
@Arlodotexe I mean we can keep that as a separate PR if we want. We just need to remove the `global using CommunityToolkit.Labs.WinUI;` line from the global file.

And maybe add it as an explicit using elsewhere in files if needed.

**Comment by michael-hawker** [comment]
Hmmm....
```
 D:\a\Labs-Windows\Labs-Windows\components\Shimmer\tests\ExampleShimmerTestClass.cs(17,31): error CS0246: The type or namespace name 'Shimmer' could not be found (are you missing a using directive or an assembly reference?) [D:\a\Labs-Windows\Labs-Windows\common\ProjectHeads\AllComponents\Tests.WinAppSdk\CommunityToolkit.Tests.WinAppSdk.csproj]
```

**Review by michael-hawker — APPROVED** [review]

**Pull request merged**